### PR TITLE
fix(social-insurance-administration): Moved shared sia messages to core

### PR DESCRIPTION
**Pull request by veronikasif**
## What

Moved shared messages to core file for all social insurance administration application. 

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

**Review by karenbjorg — COMMENTED** [review]
lftm

**Review by karenbjorg — APPROVED** [review]

**Review by ylfahfa — APPROVED** [review]
lgtm

File: libs/application/types/src/lib/ApplicationTypes.ts [hunk 1]
```diff
@@ -253,11 +253,11 @@ export const ApplicationConfigurations = {
   },
   [ApplicationTypes.OLD_AGE_PENSION]: {
     slug: 'ellilifeyrir',
-    translation: 'oap.application',
+    translation: ['oap.application', 'sia.application', 'uiForms.application'],
   },
   [ApplicationTypes.HOUSEHOLD_SUPPLEMENT]: {
     slug: 'heimilisuppbot',
-    translation: 'hs.application',
+    translation: ['hs.application', 'sia.application', 'uiForms.application'],
```
Comment (Toti91) [inline libs/application/types/src/lib/ApplicationTypes.ts:260]: Why is this change needed? 🤔

Comment (karenbjorg) [reply libs/application/types/src/lib/ApplicationTypes.ts:260]: so the messages that we are using from these namespaces are translated. 
I did this PR so this would work: https://github.com/island-is/island.is/pull/13181

Comment (Toti91) [reply libs/application/types/src/lib/ApplicationTypes.ts:260]: Ah ok, thanks 😄

**Review by Toti91 — APPROVED** [review]
Code owned LGTM
